bus-mapping/src/circuit_input_builder.rs

```diff
@@ -550,15 +560,28 @@ impl<P: JsonRpcClient> BuilderClient<P> {
     pub async fn gen_inputs(
         &self,
         block_num: u64,
+        txs_rlp: Option<&eth_types::Bytes>,
+        enable_skipping_invalid_tx: bool,
```

Comment:
Do we still need `enable_skipping_invalid_tx`? I assume if you want to pass in the raw transactions using `txs_rlp` it already means some transactions will be missing and so invalid?

Reply:
Good point. Remove `enable_skipping_invalid_tx` here.